src/format.rs

```diff
@@ -281,6 +281,11 @@ impl FormatBuilder {
         FormatBuilder { format: Box::new(format) }
     }
 
+    /// Consumes the builder and returns the generated `TableFormat`
+    pub fn into_table_format(self) -> TableFormat {
+        *self.format
+    }
```

Comment:
And here I think `into_table_format` and `from_format` are redundant since there are `From::from` and `Into::into` already...